src/services/rfqm_service.ts

```diff
@@ -262,6 +262,19 @@ export class RfqmService {
             return RfqmJobStatus.FailedValidationNoFee;
         }
 
+        if (order.type === RfqmOrderTypes.V4Rfq) {
+
+            // Orders can expire if any of the following happen:
+            // 1) workers are backed up
+            // 2) an RFQM order broke during submission and the order is stuck in the queue for a long time.
+            const v4Order = order.order;
+            const expiryInMs = new BigNumber(v4Order.expiry).times(ONE_SECOND_MS);
+            console.log()
```

Comment:
oh no

Reply:
ugh!! forgot the log msg! putting it in now